test-engineering/load-tests/kubernetes-config/locust-worker-controller.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: locust-worker
+  labels:
+    name: locust-worker
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      app: locust-worker
+  template:
+    metadata:
+      labels:
+        app: locust-worker
+    spec:
+      containers:
+        - name: locust-worker
+          image: gcr.io/[PROJECT_ID]/locust-tasks:[LOCUST_IMAGE_TAG]
+          env:
+            - name: LOCUST_MODE_WORKER
+              value: "true"
+            - name: LOCUST_MASTER_NODE_HOST
+              value: [LOCUST_MASTER_NODE_HOST]
```

Comment:
```suggestion
              value: locust-master
```

Reply:
Let's just hardcode this and delete the `LOCUST_MASTER_NODE_HOST` in test-engineering/load-tests/load_test_setup.sh#L44 since this isn't configurable.